framework/src/base/MooseApp.C

```diff
@@ -170,6 +170,7 @@ MooseApp::MooseApp(const std::string & /*deprecated_name*/, InputParameters para
     _check_input(getParam<bool>("check_input")),
     _restartable_data(libMesh::n_threads())
 {
+  srand(processor_id());
```

Comment:
Are you having a conversation with yourself again? Lol
On Fri, Jul 31, 2015 at 5:15 PM Derek Gaston <notifications@github.com>
wrote:

> In framework/src/base/MooseApp.C
> <https://github.com/idaholab/moose/pull/5491#discussion_r36024943>:
>
> > @@ -170,6 +170,7 @@ MooseApp::MooseApp(const std::string & /*deprecated_name*/, InputParameters para
> >      _check_input(getParam<bool>("check_input")),
> >      _restartable_data(libMesh::n_threads())
> >  {
> > +  srand(processor_id());
>
> If we're removing it - why is it here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5491/files#r36024943>.
>